### PR TITLE
Add the github branch as a label

### PR DESCRIPTION
**Pull request by lukebjerring**
## Description
This will allow us to distinguish `master` runs.

Fixes https://github.com/web-platform-tests/wpt.fyi/issues/699

### CODE REVIEW

File: api/webhook/taskcluster.go [hunk 1]
```diff
@@ -88,8 +88,34 @@ type statusEventPayload struct {
...
+
 type branchInfo struct {
-	Name string `json:"name"`
+	Name   string     `json:"name"`
+	Commit commitInfo `json:"commit"`
+}
+
+type commitInfo struct {
+	Sha string `json:"sha"`
+	URL string `json:"url"`
```
Comment (Hexcles) [inline api/webhook/taskcluster.go]: Do we need the URL of a commit? If not, let's remove this field. The API payload from both GitHub and Taskcluster are big and complicated, so I like to include only the necessary fields in various structs, which simplifies the struct definition and avoids distraction.

File: api/webhook/taskcluster.go [hunk 2]
```diff
@@ -88,8 +88,34 @@ type statusEventPayload struct {
 	Branches  []branchInfo `json:"branches"`
 }
 
+// GetBranch returns the first branch that has the revision of the commit
+// as its HEAD state.
+func (s statusEventPayload) GetBranch() string {
```
Comment (Hexcles) [inline api/webhook/taskcluster.go]: Using this to decide whether a status update comes from the master branch might be problematic.

IIUC, the `branches` field contains a list of branches that contain this commit, each one with its *current* head. When a Taskcluster run finishes for a commit and posts the "success" status, that commit is very likely no longer the head of master.

I think we need two methods, like `IsOnMaster` and `HeadingBranches`, and only the latter looks at the SHA and will be used in the PR case.

File: api/webhook/taskcluster.go [hunk 3]
```diff
@@ -88,8 +88,34 @@ type statusEventPayload struct {
...
+}
+
 type branchInfo struct {
-	Name string `json:"name"`
+	Name   string     `json:"name"`
+	Commit commitInfo `json:"commit"`
+}
+
+type commitInfo struct {
+	Sha string `json:"sha"`
```
Comment (Hexcles) [inline api/webhook/taskcluster.go]: Nit: SHA instead of Sha.

File: api/webhook/taskcluster.go (hunk 1 above)
Comment (lukebjerring) [reply api/webhook/taskcluster.go]: Done.

**Pull request merged**